sources/core/objects/Interface.cpp

```diff
@@ -128,23 +138,23 @@ void HumanInterface::UpdateHealth()
     if (IdPtr<Human> human = owner_)
     {
         int health = human->GetHealth();
-        if (health == 100)
+        if (health == 10000)
         {
             health_.GetView()->SetState("health0");
         }
-        else if (health >= 80)
```

Comment:
It is old bad code, so it should be rewritten a little bit.
```
const int STATES_AMOUNT = 7;
int state = (health * STATES_AMOUNT) / (max_health + 1); // it will be from 0 to 6
state = STATES_AMOUNT - state - 1; // 0 to 6, but reverted
health_.GetView()->SetState(QString("health%1").arg(state));
```